arq/connections.py

```diff
@@ -47,6 +47,10 @@ class RedisSettings:
     sentinel: bool = False
     sentinel_master: str = 'mymaster'
 
+    retry_on_timeout: bool = False
+    retry_on_error: Optional[List[Any]] = None
+    retry: Optional[Any] = None
```

Comment:
This isn't an `Any`, but rather a `redis.Retry`, right?

Reply:
yup, fixed.